ui/client/reducers/settings.ts

```diff
@@ -12,8 +18,8 @@ export type SettingsState = {
 }
 
 export type AuthenticationSettings = {
-  backend: string,
-  authorizeUrl?: URL,
+  backend: AuthBackends,
+  authorizeUrl?: string,
```

Comment:
Why not URL?

Reply:
to avoid changes outside frontend part. when settings/props will change this will also change.